scripts.js

```diff
@@ -1,65 +1,92 @@
+    } else {
+      const bookStorage = localStorage.getItem('books');
+      const bookArray = JSON.parse(bookStorage);
+      bookArray.push(book);
+      localStorage.setItem('books', JSON.stringify(bookArray));
+    }
+
+    title.value = '';
+    author.value = '';
+    this.dotiBooks();
+  }
+
+  dotiBooks() {
+    const wrapper = document.createElement('div');
     const hline = document.createElement('hr');
-    displayTitle.innerText = element.title;
-    showAuthor.innerText = element.author;
-    delbtn.innerHTML = `<button onclick='removeBook(${index})'>Remove</button>`;
-    delbtn.classList.add('deleteBook');
-    container.appendChild(displayTitle);
-    container.appendChild(showAuthor);
-    container.appendChild(delbtn);
-    container.appendChild(hline);
-    wrapper.appendChild(container);
-  });
-  showArea.appendChild(wrapper);
-}
-/* Take the data store it  */
-const storeObj = {};
-function dataStorage() {
-  if (localStorage.getItem('books') === null) {
-    const bookStore = [];
-    bookStore.push(storeObj);
-    localStorage.setItem('books', JSON.stringify(bookStore));
-  } else {
     const bookStorage = localStorage.getItem('books');
+    const dotion = document.createElement('div');
+    dotion.innerText = this.author;
     const bookArray = JSON.parse(bookStorage);
-    bookArray.push(storeObj);
+    bookArray.forEach((element, index) => {
+      const displayTitle = document.createElement('p');
+      const displayAuthor = document.createElement('p');
+      const delbtn = document.createElement('div');
+      const container = document.createElement('div');
+      const senti = document.createElement('div');
+      displayTitle.innerText = `"${element.title}" by`;
+      displayAuthor.innerText = element.author;
+      delbtn.innerHTML = `<button class="btn outer" onclick='deleteItem(${index})'>Remove</button>`;
+      delbtn.classList.add('deleteBook');
+      container.classList.add('library', 'middle');
+      senti.classList.add('library');
+      displayAuthor.classList.add('letter');
+      senti.appendChild(displayTitle);
+      senti.appendChild(displayAuthor);
+      container.appendChild(senti);
+      container.appendChild(delbtn);
+      wrapper.appendChild(container);
+    });
+    hline.classList.add('hline');
+    showArea.appendChild(wrapper);
+    showArea.appendChild(hline);
+  }
+
+  removeBook(index) {
+    const bookStorage = localStorage.getItem('books');
+    const bookArray = JSON.parse(bookStorage);
+    bookArray.splice(index, 1);
     localStorage.setItem('books', JSON.stringify(bookArray));
+    showArea.innerHTML = '';
+    this.dotiBooks();
   }
-  title.value = '';
-  author.value = '';
-  dotiBooks();
 }
+
 form.addEventListener('submit', (event) => {
   event.preventDefault();
-  storeObj.title = title.value;
-  storeObj.author = author.value;
-  showArea.innerHTML = '';
-  dataStorage();
+  if (title.value === '') {
+    alert('Please enter a book title');
```

Comment:
- [ ] Next, related to alert 😜, according to [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) (👈 this is a link), it is not advised to use alerts in code. To fix this issue kindly remove it 🚮 and replace it with more suitable alternatives or take the time to create something new and exciting 😊.

Reply:
Thank you for the feedback